### PR TITLE
Operator further coverage

### PR DESCRIPTION
**Pull request by samigabor**
Only  `_msgData()` remained to be covered

### CODE REVIEW

File: packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts [hunk 1]
```diff
@@ -271,6 +271,68 @@ describe("Operator contract", (): void => {
...
         })
+
+        it("only operators can reduce the stake", async function(): Promise<void> {
+            const { token } = sharedContracts
+            setTokens(delegator, "100")
+            const { operator } = await deployOperator(operatorWallet)
+            await (await token.connect(delegator).transferAndCall(operator.address, parseEther("100"), "0x")).wait()
+            const sponsorship = await deploySponsorship(sharedContracts)
+            await operator.stake(sponsorship.address, parseEther("100"))
+
```
Comment (jtakalai) [inline packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts:282]: you could deploy a "defaultOperator" in the `before` and re-use it for non-mutating tests (access checks etc)

File: packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts [hunk 2]
```diff
@@ -448,6 +510,52 @@ describe("Operator contract", (): void => {
                 await (await sharedContracts.streamrConfig.setMinimumSelfDelegationFraction("0")).wait()
             })
 
+            it("reverts on set delegation policy since only the factory should be able to set it at deploy time", async function(): Promise<void> {
+                const { operator } = await deployOperator(operatorWallet)
```
Comment (jtakalai) [inline packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts:514]: these could also use the defaultOperator

File: packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts [hunk 3]
```diff
@@ -1602,6 +1710,21 @@ describe("Operator contract", (): void => {
     })
 
     describe("Operator/owner", () => {
+        it("reverts if trying to call initialize()", async function(): Promise<void> {
+            const { operator } = await deployOperator(operatorWallet)
+            const { token, streamrConfig, nodeModule, queueModule, stakeModule } = sharedContracts
+            await expect(operator.initialize(
```
Comment (jtakalai) [inline packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts:1716]: this could be defaultOperator as well

**Review by jtakalai — CHANGES_REQUESTED** [review]

File: packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts (hunk 1 above)
Comment (samigabor) [reply packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts]: Updated

File: packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts (hunk 3 above)
Comment (samigabor) [reply packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts]: updated

File: packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts (hunk 2 above)
Comment (samigabor) [reply packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts]: updated

**Pull request merged**